pr2_gazebo_plugins/CMakeLists.txt

```diff
@@ -15,10 +15,8 @@ endif()
 
 # Depend on system install of Gazebo and SDFormat
 find_package(gazebo REQUIRED)
-pkg_check_modules(SDF sdformat REQUIRED)
```

Comment:
[The package still uses sdformat](https://github.com/PR2/pr2_simulator/blob/kinetic-devel/pr2_gazebo_plugins/src/gazebo_ros_power_monitor.cpp#L33), relying on the SDFormat include.
Does Gazebo add the header include or why do you just remove it here? Should we switch to a `find_package` (which SDFormat provides for years by now)?

Reply:
this issue is resolved in pull request #149